docs-private/Developer-How-To-Start.md

```diff
@@ -6,3 +6,31 @@
 - Enable maven profile `standalone`
 - Use IntelliJ Idea run configuration at `../.run/UserDataStoreApplication.run.xml`
 - Open [http://localhost:8091/user-data-store/actuator/health](http://localhost:8091/user-data-store/actuator/health) and you should get `{"status":"UP"}`
+
+
+## Database
+
+Database changes are driven by Liquibase.
+
+This is an example how to manually check the Liquibase status.
+Important and fixed parameter is `changeloge-file`.
+Others (like URL, username, password) depend on your environment.
+Mind that the working directory MUST be `src/main/resources` to be consistent with Spring Boot naming (file names are part of the checksum).
```

Comment:
I think no need to capitalize `MUST`.

Reply:
Ok, too strict. I was thinking about RFC 2119